### DISCUSSION TITLE
¿Cuanto es lo maximo que se puede subir a github en tamaño de almacenamiento?

### DISCUSSION THREAD

**Discussion by huerito23** [community/community#66057] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

¿Cuanto es lo maximo que se puede subir a github en tamaño de almacenamiento?

**Comment by ozuro** [comment]
Los repositorios de cuentas gratuitas tenían un límite de 100 MB por archivo y un límite total de 1 GB para el repositorio.

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
